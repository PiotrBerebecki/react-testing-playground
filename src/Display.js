import React from 'react';

function Display({ count }) {
  return (
    <div className="count">
      Current count: {count}
    </div>
  );
}

export default Display;
