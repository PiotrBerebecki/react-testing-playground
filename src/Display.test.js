import React from 'react';
import renderer from 'react-test-renderer';

import Display from './Display';

describe('Display', () => {
  test('Display renders correctly', () => {
    const tree = renderer.create(<Display count={22} />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
