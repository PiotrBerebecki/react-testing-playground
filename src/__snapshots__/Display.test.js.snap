exports[`Display Display renders correctly 1`] = `
<div
  className="count">
  Current count: 
  22
</div>
`;
